cogs/moderation.py

```diff
@@ -20,91 +129,216 @@ def __init__(self, bot: Bot) -> None:
 
-        await ctx.send(embed=embed)
-        await member.send(embed=embed)
+                    **❯❯ You can only kick server members.**
+                    """
+                ),
+                color=discord.Color.red(),
+            )
+            await ctx.send(f"Sorry {ctx.author.mention}", embed=embed)
+            return
+
+        server_embed = discord.Embed(
+            title="User Kicked",
+            description=textwrap.dedent(
+                f"""
+                **Reason**: {reason}
+                **User**: {member.mention} (`{member.id}`)
+                **Actor**: {ctx.author.mention} (`{ctx.author.id}`)
+                """
+            ),
+            color=discord.Color.orange(),
+            timestamp=datetime.utcnow(),
+        )
+        server_embed.set_thumbnail(url=member.avatar_url_as(format="png", size=256))
+
+        dm_embed = discord.Embed(
+            title="You were Kicked",
+            description=textwrap.dedent(
+                f"""
+                {reason}
+
+                *Server: {ctx.guild.name}*
+                """
+            ),
+            color=discord.Color.red(),
+            timestamp=datetime.utcnow(),
+        )
+        dm_embed.set_thumbnail(url=ctx.guild.icon_url)
+
+        await ctx.send(embed=server_embed)
+        await member.send(embed=dm_embed)
         await member.kick(reason=reason)
 
     @command()
-    @bot_has_permissions(ban_members=True)
     @has_permissions(ban_members=True)
-    async def ban(self, ctx: Context, member: discord.Member, *, reason: str = "No Reason Stated.") -> None:
+    @follow_roles()
+    async def ban(self, ctx: Context, member: UserID, *, reason: str = "No specific reason.") -> None:
         """Ban a User."""
-        embed = discord.Embed(title="Infraction information", color=discord.Color.red())
-        embed.add_field(name="Type", value="Ban")
-        embed.add_field(name="Reason", value=reason)
-        embed.set_thumbnail(url=member.avatar_url)
-        embed.set_author(name=member.name, url=member.avatar_url)
-        embed.set_footer(text=member.guild.name, icon_url=member.guild.icon_url)
+        server_embed = discord.Embed(
+            title="User Banned",
+            description=textwrap.dedent(
+                f"""
+                **Reason**: {reason}
+                **User**: {member.mention} (`{member.id}`)
+                **Actor**: {ctx.author.mention} (`{ctx.author.id}`)
+                """
+            ),
+            color=discord.Color.orange(),
+            timestamp=datetime.utcnow(),
+        )
+        server_embed.set_thumbnail(url=member.avatar_url_as(format="png", size=256))
 
-        await ctx.send(embed=embed)
-        await member.send(embed=embed)
+        dm_embed = discord.Embed(
+            title="You were Banned",
+            description=textwrap.dedent(
+                f"""
+                {reason}
+
+                *Server: {ctx.guild.name}*
```

Comment:
```suggestion
                **Server: {ctx.guild.name}**
```

Reply:
No, I like this to only be italic, not bold, that's bringing too much attention on Server and doesn't look good, I've tried it. The reason should be the important factor in the embed, not the server.